### PR TITLE
create user view

### PR DESCRIPTION
**Pull request by BernardoTobon**
we want replicate this view 

- [x] navigation tab.
- [x] profile.
- [x] user history
- [x] life style
- [x] patient goal
- [x] recommendations

image

### CODE REVIEW

File: src/components/Data.js [hunk 1]
```diff
@@ -0,0 +1,348 @@
...
+                    stroke="currentColor"
+                    className="w-6 h-6"
+                  >
+                    <path
+                      strokeLinecap="round"
+                      strokeLinejoin="round"
+                      d="M10.5 1.5H8.25A2.25 2.25 0 006 3.75v16.5a2.25 2.25 0 002.25 2.25h7.5A2.25 2.25 0 0018 20.25V3.75a2.25 2.25 0 00-2.25-2.25H13.5m-3 0V3h3V1.5m-3 0h3m-3 18.75h3"
+                    />
+                  </svg>
+                  <span className="text-light-brown">Celuklar</span>
```
Comment (josezy) [inline src/components/Data.js:219]:
```suggestion
                  <span className="text-light-brown">Celular</span>
```

File: src/components/Data.js [hunk 2]
```diff
@@ -0,0 +1,348 @@
...
+                    stroke="currentColor"
+                    className="w-6 h-6"
+                  >
+                    <path
+                      strokeLinecap="round"
+                      strokeLinejoin="round"
+                      d="M2.25 6.75c0 8.284 6.716 15 15 15h2.25a2.25 2.25 0 002.25-2.25v-1.372c0-.516-.351-.966-.852-1.091l-4.423-1.106c-.44-.11-.902.055-1.173.417l-.97 1.293c-.282.376-.769.542-1.21.38a12.035 12.035 0 01-7.143-7.143c-.162-.441.004-.928.38-1.21l1.293-.97c.363-.271.527-.734.417-1.173L6.963 3.102a1.125 1.125 0 00-1.091-.852H4.5A2.25 2.25 0 002.25 4.5v2.25z"
+                    />
+                  </svg>
+                  <span className="text-light-brown">Telefono Fijo</span>
```
Comment (josezy) [inline src/components/Data.js:242]:
```suggestion
                  <span className="text-light-brown">Teléfono Fijo</span>
```

File: src/components/Data.js [hunk 3]
```diff
@@ -0,0 +1,348 @@
...
+                    stroke="currentColor"
+                    className="w-6 h-6"
+                  >
+                    <path
+                      strokeLinecap="round"
+                      strokeLinejoin="round"
+                      d="M12.75 3.03v.568c0 .334.148.65.405.864l1.068.89c.442.369.535 1.01.216 1.49l-.51.766a2.25 2.25 0 01-1.161.886l-.143.048a1.107 1.107 0 00-.57 1.664c.369.555.169 1.307-.427 1.605L9 13.125l.423 1.059a.956.956 0 01-1.652.928l-.679-.906a1.125 1.125 0 00-1.906.172L4.5 15.75l-.612.153M12.75 3.031a9 9 0 00-8.862 12.872M12.75 3.031a9 9 0 016.69 14.036m0 0l-.177-.529A2.25 2.25 0 0017.128 15H16.5l-.324-.324a1.453 1.453 0 00-2.328.377l-.036.073a1.586 1.586 0 01-.982.816l-.99.282c-.55.157-.894.702-.8 1.267l.073.438c.08.474.49.821.97.821.846 0 1.598.542 1.865 1.345l.215.643m5.276-3.67a9.012 9.012 0 01-5.276 3.67m0 0a9 9 0 01-10.275-4.835M15.75 9c0 .896-.393 1.7-1.016 2.25"
+                    />
+                  </svg>
+                  <span className="text-light-brown">Direccion</span>
```
Comment (josezy) [inline src/components/Data.js:287]:
```suggestion
                  <span className="text-light-brown">Dirección</span>
```

File: src/components/TabMenu.js
Comment (josezy) [inline src/components/TabMenu.js:1]: the file extension `.js` is meant for normal JS code, if the file contains JSX elements (any code like `<></>`), the file extension should be `.jsx`

File: src/components/UserHistory.js [hunk 4]
```diff
@@ -0,0 +1,213 @@
+import UserTabs from "./TabMenu";
+
+const HistoryUser = () => {
+  return (
+    <>
+      <UserTabs>
+        <div className="flex items-center flex-col">
+          <h1 className="flex justify-center m-3 flex justify-center font-extrabold  text-transparent text-2xl bg-clip-text bg-gradient-to-r from-light-brown to-dark-green">
+            Historia Clinica
```
Comment (josezy) [inline src/components/UserHistory.js:9]:
```suggestion
            Historia Clínica
```

File: src/components/UserHistory.js [hunk 5]
```diff
@@ -0,0 +1,213 @@
+import UserTabs from "./TabMenu";
+
+const HistoryUser = () => {
```
Comment (josezy) [inline src/components/UserHistory.js:3]: be consistent with naming, filename is `UserHistory.js` while the component is called `HistoryUser`.

`UserHistory` is grammatically correct

**Review by josezy — COMMENTED** [review]
Good work overall, some notes:

- I see tons of SVG code for icons, in general, those are stored as custom files, i.e. the svg code for a 'gear' icon is a component in a file called `icons/Gear.jsx`, and you can reuse them!

- Your code looks clean, but set up and use a formatter to achieve a cooler code style

💪

File: src/components/TabMenu.js
Comment (abcei2) [reply src/components/TabMenu.js:1]: :O

**Pull request merged**